test/DynamoCoreTests/ProfilingTest.cs

```diff
@@ -71,5 +71,28 @@ public void TestProfilingSingleNode()
             engineController.EnableProfiling(false, homeWorkspace, nodes);
             Assert.IsNull(engineController.ProfilingSession);
         }
+ 
+        [Test]
+        public void TestProfilingSingleNodePublicMethodsOnly()
```

Comment:
I feel this test is redundant; it's testing what is already being tested in the first test case you added, is it not?

Reply:
It is a little redundant, but it is testing the public data accessor as opposed to the other test that is using internal methods.